### ISSUE TITLE
Binary errors with /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.29' for node:16.14.2 Docker base image

### ISSUE THREAD

**Issue by zachsa** [guybedford/chomp#117]
in a Dockerfile using `Node 16.14.2` as the base image:

```Dockerfile
FROM node:16.14.2
RUN npm install -g chomp
RUN chomp --version
```

I get an error:

```
/usr/local/lib/node_modules/chomp/vendor/chomp: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.29' not found (required by /usr/local/lib/node_modules/chomp/vendor/chomp)
```

This doesn't occur when using a later Node base image (`FROM node:18.0.0`), which is great. But using chomp with node v16.14.2 is tricky. Would it be possible to update the binary to support this use case?

**Comment by JayaKrishnaNamburu** [comment]
Hey @zachsa the issue was addressed in #118. Please try it

**Comment by zachsa** [comment]
Hi,

Thank you - I can see that Chomp 0.2.3 is working in containers created from a node:16.14.2 image now
